src/protocol/frame.rs

```diff
@@ -163,7 +163,7 @@ mod tests {
 
         data.set_position(0);
         let actual = data.read_message(0).await.unwrap();
-        assert_eq!(actual, vec![]);
+        assert!(actual.is_empty())
```

Comment:
why was that changed?

Reply:
I have no clue why, but on newer rust versions the old approach fails to compile due to type resolution.

Reply:
OK, then that's fine :slightly_smiling_face: